R-package/src/install.libs.R

```diff
@@ -67,6 +67,7 @@ if (!use_precompile) {
   # Check if Windows installation (for gcc vs Visual Studio)
   if (WINDOWS) {
     if (use_mingw) {
+      print("Trying to build with MinGW")
```

Comment:
Change in another PR `print` for `message`.

Reply:
sure! Let's talk about it in a separate issue / PR. We have several other `print()` statements in `install.libs.R` and no uses of `message()` so I'm curious what makes `message()` preferable here.